digits/model/tasks/caffe_train.py

```diff
@@ -1128,18 +1123,26 @@ def get_task_stats(self,epoch=-1):
 
         loc, mean_file = os.path.split(self.dataset.get_mean_file())
 
+        image_dimensions = list(self.dataset.get_feature_dims())
+
+        # TODO: Store both original, and cropped image dimensions
+        if self.crop_size is not None:
+            image_dimensions[0] = self.crop_size
+            image_dimensions[1] = self.crop_size
+
         stats = {
-            "image dimensions": self.dataset.get_feature_dims(),
+            "image dimensions": tuple(image_dimensions),
```

Comment:
on below line, this way of getting the mean file doesn't work for generic datasets (mean file is not located in root of job directory but in `train_db` sub-folder).